### PR TITLE
Add links to meetup and slack group

### PR DESCRIPTION
**Pull request by MgenGlder**
References https://github.com/dearborn-coding-club/website-base-backend/issues/24

## Summary
- Adds reference links for slack and meetup.

### CODE REVIEW

File: README.md
Comment (dXu23) [inline README.md:1]: Not sure if it's intentional, but the slack channel and meetup group have the same links.

**Review by dXu23 — COMMENTED** [review]
As I mentioned in the comment, it's the same link for both the slack channel and meetup group. I'm wondering if this needs to be changed?

**Comment by MgenGlder** [comment]
> As I mentioned in the comment, it's the same link for both the slack channel and meetup group. I'm wondering if this needs to be changed?

Ahh yep! Thanks- good catch. Just updated.

**Review by dXu23 — APPROVED** [review]
LGTM

**Pull request merged**